CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## Unreleased
 
-## 2.12.0 (2023-11-14)
```

Comment:
This should have been fixed because the version was published with some real changes. Removing it is not the right thing. I added a fix commit to master, but be sure to resolve your conflict here correctly.